index.js

```diff
@@ -73,11 +73,12 @@ const proxyHandler = {
   set (obj, prop, value) {
     if (prop.slice(0, 2) === '__') {
       obj[prop] = value
+      return true
```

Comment:
Also, just confirmed locally that this entire `set` method can just throw the error in all cases, since the `FiggyPudding` constructor appears to set the `__`-prefixed properties before the `Proxy` gets their paws on the instance.

```diff
diff --git a/index.js b/index.js
index a81b442..3b1ba1c 100644
--- a/index.js
+++ b/index.js
@@ -71,12 +71,7 @@ const proxyHandler = {
     return obj.get(prop)
   },
   set (obj, prop, value) {
-    if (prop.slice(0, 2) === '__') {
-      obj[prop] = value
-      return true
-    } else {
-      throw new Error('figgyPudding options cannot be modified. Use .concat() instead.')
-    }
+    throw new Error('figgyPudding options cannot be modified. Use .concat() instead.')
   },
   deleteProperty () {
     throw new Error('figgyPudding options cannot be deleted. Use .concat() and shadow them instead.')
diff --git a/test/index.js b/test/index.js
index 4c0da2e..3512c0e 100644
--- a/test/index.js
+++ b/test/index.js
@@ -173,13 +173,13 @@ test('proxy', t => {
   t.equal('b' in opts, false, 'missing opt undetected')
   t.equal('c' in opts, false, 'undefined opt undetected')
   t.equal(opts.a, 1, 'provided key retrieved')
-  t.doesNotThrow(
+  t.throws(
     () => {
       opts.__woo = 'woo'
     },
-    'setting private keys is possible'
+    /figgyPudding options cannot be modified/i,
+    'setting private keys is forbidden'
   )
-  t.equal(opts.__woo, 'woo', 'private key retrieved')
   t.throws(
     () => {
       opts.b = 'yo'
```